packages/rancher-vsphere/rancher-vsphere-cpi/generated-changes/patch/Chart.yaml.patch

```diff
@@ -1,11 +1,15 @@
 --- charts-original/Chart.yaml
 +++ charts/Chart.yaml
-@@ -5,7 +5,7 @@
+@@ -5,10 +5,10 @@
    catalog.cattle.io/namespace: kube-system
    catalog.cattle.io/os: linux
    catalog.cattle.io/permits-os: linux,windows
```

Comment:
Were these changes made manually? PR looks fine in general, just curious since I haven't seen the patch file being updated in the same commit as the package.yaml file changes

Reply:
No that was generated automatically